### ISSUE TITLE
Couldn’t sign you in

### ISSUE THREAD

**Issue by bociana** [klaussinani/tusk#338]
Hi,  I have problem to sign in.

When i try using my google account:

"Couldn’t sign you in
This browser or app may not be secure. Learn more

Try using a different browser. If you’re already using a supported browser, you can refresh your screen and try again to sign in."

Ubuntu 18.04.5 LTS

**Comment by suzdalnitski** [comment]
Same problem here...

**Comment by carrotfarmer** [comment]
Any fix for this? I tried uninstalling to using 
```
yay -R tusk
```

but it still shows the same error after reinstallation :/

**Comment by brauntek** [comment]
Nope, doesn't work for me. I don't like the Evernote Web Package. Nix is the the only option, next to opening the website every darn time.
